docs/api/ipc-main.md

```diff
@@ -90,8 +90,14 @@ Set this to the value to be returned in a synchronous message.
 
 ### `event.sender`
 
-Returns the `webContents` that sent the message, you can call
+Returns the `ipcRenderer` that sent the message, you can call
```

Comment:
@paulcbetts hmm, I think it should be the `webContents`.

This is what I tried in the main process:

```js
require('electron').ipcMain.on('foo', (event) => {
  console.log(event.sender.getId());
  console.log(mainWindow.webContents.getId());
  console.log(event.sender === mainWindow.webContents);
})
```

And then in the dev tools Console:

```js
require('electron').ipcRenderer.send('foo')
```

And I see the following printed:

```
17179869186
17179869186
true
```

What were you seeing not being available?